script.js

```diff
@@ -0,0 +1,32 @@
+console.log("script linked!");
+
+document.turn = "X";
+
+function startGame() {
+}
+
+setMessage(document.turn + " " + " U Go first");
```

Comment:
I expected this to be called within `startGame`, since this the message displayed at the start of the game.  :)